src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ticketRuleModal.tsx

```diff
@@ -49,10 +49,21 @@ class TicketRuleModal extends AbstractExternalIssueForm<Props, State> {
 
   getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
     const {instance, organization} = this.props;
+    const query = ((instance.dynamic_form_fields as {[key: string]: any}) || [])
```

Comment:
@mgaeta Instead of type casting here, we should try to type `dynamic_form_fields` correctly. Maybe we need to add the `dynamic_form_fields` field to the type of `instance`.

Reply:
👍🏽 I'll update the types